ui/src/containers/AppsTableContainer.js

```diff
@@ -1,57 +1,47 @@
 import React from 'react';
-import { sortable, SortByDirection } from '@patternfly/react-table';
+import { SortByDirection } from '@patternfly/react-table';
 import AppsTable from '../components/AppsTable';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { getApps } from '../actions/actions';
+import { reverseAppsTableSort } from '../actions/actions-ui';
 
-class AppsTableContainer extends React.Component {
+export class AppsTableContainer extends React.Component {
   constructor (props) {
     super(props);
-    this.sortBy = {};
     this.onSort = this.onSort.bind(this);
     this.onRowClick = this.onRowClick.bind(this);
   }
-  componentDidMount () {
-    this.props.getApps();
-  }
   onRowClick (event, rowId, props) {
+    console.log('On row click called');
```

Comment:
Is this method needed? I don't think we use it or plan to use it

Reply:
@damienomurchu we'll use it later for navigation to the AppDetailedView, which should be covered in this [JIRA](https://issues.jboss.org/browse/AEROGEAR-8552)

Reply:
I can remove it for now if you like but it would get added back in later I imagine, unless we go with a different approach for navigating. I think though based on the way the table structure is wrapped up internally in the Patternfly table this is probably the easiest approach.

Reply:
@laurafitzgerald perhaps leave it in for now then, as I see what you mean. Let's see how we implement the routing/ redirect to the UI detail page when an app is clicked on

Reply:
@damienomurchu 👍 I was thinking about taking that one next. Will resolve this conversation for now.